tests/integration/targets/anycast_config/tasks/main.yml

```
@@ -0,0 +1,209 @@
+          service: "DNS"
+          name: "{{ ac_config_name }}"
+        register: ac_config_info
+      - assert:
+          that:
+              - ac_config is changed
+              - ac_config_info is not failed
+              - ac_config_info.objects | length == 1
+              - ac_config_info.objects[0].description == "Test description for Anycast configuration"
+
+      - name: Create Anycast Configuration with Tags
+        infoblox.universal_ddi.anycast_config:
+          name: "{{ ac_config_name }}"
+          anycast_ip_address: "10.0.0.0"
+          service: "DNS"
+          tags:
+            location: "site-1"
+          state: "present"
+        register: ac_config
+      - name: Get Anycast Configuration Information
+        infoblox.universal_ddi.anycast_config_info:
+          service: "DNS"
+          name: "{{ ac_config_name }}"
+        register: ac_config_info
+      - assert:
+          that:
+            - ac_config is changed
+            - ac_config_info is not failed
+            - ac_config_info.objects | length == 1
+            - ac_config_info.objects[0].tags.location == "site-1"
+
+      - name: Create an Anycast Configuration with a different Anycast IP Address
+        infoblox.universal_ddi.anycast_config:
+          name: "{{ ac_config_name }}"
+          anycast_ip_address: "20.0.0.0"
+          service: "DNS"
+          state: "present"
+        register: ac_config
+      - name: Get Anycast Configuration Information
+        infoblox.universal_ddi.anycast_config_info:
+          service: "DNS"
+          name: "{{ ac_config_name }}"
+        register: ac_config_info
+      - assert:
+          that:
+              - ac_config is changed
+              - ac_config_info is not failed
+              - ac_config_info.objects | length == 1
+              - ac_config_info.objects[0].anycast_ip_address == "20.0.0.0"
+
+      - name: Create a new Anycast Configuration with a different Service
+        infoblox.universal_ddi.anycast_config:
+          name: "{{ ac_config_name_1 }}"
+          anycast_ip_address: "10.10.0.0"
+          service: "DHCP"
+          state: "present"
+        register: ac_config
+      - name: Get Anycast Configuration Information
+        infoblox.universal_ddi.anycast_config_info:
+          service: "DHCP"
+          name: "{{ ac_config_name_1 }}"
+        register: ac_config_info
+      - assert:
+          that:
+              - ac_config is changed
+              - ac_config_info is not failed
+              - ac_config_info.objects | length == 1
+              - ac_config_info.objects[0].name == ac_config_name_1
+              - ac_config_info.objects[0].service == "DHCP"
+
+  always:
+    - name: Delete Anycast Configuration
+      infoblox.universal_ddi.anycast_config:
+        name: "{{ ac_config_name }}"
+        anycast_ip_address: "20.0.0.0"
+        service: "DNS"
+        state: absent
+      ignore_errors: true
+
+    - name: Delete Anycast Configuration
```

```suggestion
    - name: Delete Anycast Configuration enabled with DHCP Service
```
Generally a good idea to differentiate Task names.

---

Done